[{"name": "test.test_inverse_series.InverseSeries.test_called", "status": "passed", "message": "", "passed": true, "points": ["p03-15.1"], "backtrace": []}, {"name": "test.test_inverse_series.InverseSeries.test_empty", "status": "passed", "message": "", "passed": true, "points": ["p03-15.1"], "backtrace": []}, {"name": "test.test_inverse_series.InverseSeries.test_first", "status": "passed", "message": "", "passed": true, "points": ["p03-15.1"], "backtrace": []}, {"name": "test.test_inverse_series.InverseSeries.test_second", "status": "errored", "message": "memoryview: format O not supported", "passed": false, "points": ["p03-15.1"], "backtrace": ["  File \"C:\\ProgramData\\Anaconda3\\lib\\unittest\\case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"C:\\ProgramData\\Anaconda3\\lib\\unittest\\case.py\", line 615, in run\n    testMethod()\n", "  File \"C:\\Users\\koske\\Documents\\Koodit\\hy-data-analysis-with-python-summer-2019\\part03-e15_inverse_series\\test\\test_inverse_series.py\", line 33, in test_second\n    t = inverse_series(s)\n", "  File \"C:\\Users\\koske\\Documents\\Koodit\\hy-data-analysis-with-python-summer-2019\\part03-e15_inverse_series\\src\\inverse_series.py\", line 10, in inverse_series\n    r=pd.Series(i,index=vals)\n", "  File \"C:\\ProgramData\\Anaconda3\\lib\\site-packages\\pandas\\core\\series.py\", line 169, in __init__\n    index = ensure_index(index)\n", "  File \"C:\\ProgramData\\Anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\", line 5378, in ensure_index\n    return Index(index_like)\n", "  File \"C:\\ProgramData\\Anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\", line 475, in __new__\n    if data and all(isinstance(e, tuple) for e in data):\n", "  File \"C:\\ProgramData\\Anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\", line 475, in <genexpr>\n    if data and all(isinstance(e, tuple) for e in data):\n"]}]